packages/editor/src/components/post-text-editor/test/index.js

```diff
@@ -7,13 +7,14 @@ import Textarea from 'react-autosize-textarea';
 /**
  * WordPress dependencies
  */
-import { useSelect } from '@wordpress/data';
+import * as wp from '@wordpress/data';
```

Comment:
@grzim This looks like some leftover from before https://github.com/WordPress/gutenberg/pull/27717/commits/22598167b14a3b3c331cbf9b30a8f7ea9a4d261a? Should this be reverted?

Reply:
Yes, it should be removed, `useSelect` is mocked with `jest.mock` call in another place.